railseventstore.org/source/docs/exceptions.html.md

```diff
@@ -75,6 +75,16 @@ expect do
 end.to raise_error(InvalidPageStart)
 ```
 
+### RubyEventStore::InvalidPageStop
```

Comment:
Big ups for the docs! Ideally we want them to be visible after this code is released into a gem. 

We either mark those not-released fragments with annotation that this is unreleased feature or wrap docs code in feature toggle: https://github.com/RailsEventStore/rails_event_store/blob/master/railseventstore.org/config.rb#L23-L41. Example usage: f869949de5298b78e94ca3b1e99a9f6f7bf0895b